### PR TITLE
Fix BPMN graph creation for empty activities

### LINKED ISSUE #20: BPMN Graph Creation Fails When All Activities Are Filtered Out
## Description

The BPMN graph creation process fails when using the Inductive Miner algorithms with an SPM metric of 1.0 on datasets where all activities are filtered out or non-existent. Currently, this issue occurs with the Inductive Miner and Inductive Miner Infrequent. When a model is built where all potential activities are filtered, the system throws an error in the app and log instead of handling the edge case gracefully. It should create a valid graph containing only a Start Node and an End Activity/Task, similar to the behavior observed with the Petri Net algorithm.

## Steps to Reproduce

- Load the `test_csv.csv` file into the application.
- Select either the Inductive Miner or Inductive Miner Infrequent algorithm.
- Set the SPM metric to `1.0`.
- Observe the application view and console log after completion.

## Expected Behavior

The application should successfully generate a valid BPMN graph even when all activities are filtered out due to the SPM metric settings. The resulting graph should visually display only a Start Node and an End Node, maintaining structural integrity, and no error should be logged in the console.

## Actual Behavior

An error is thrown within the application interface, and error details appear in the log. No BPMN graph is generated in the view panel. It does not fall back to displaying a Start/End-only graph like the Petri Net  does.

### PR DESCRIPTION
**Pull request by reckmin**
## What was changed
This PR fixes issue #20
When all activities are filtered out, the converter now creates an empty BPMN graph with Start and End nodes.

## Testing
- Tested manually using scenario from issue #20 
- Ran `python -m unittest discover -p "*test.py"`

### CODE REVIEW

**Review by fabianf00 — CHANGES_REQUESTED** [review]
A unit test should be added for this specific edge case, so this bug does not occur again in the future.

File: tests/converters/bpmn_converter_test.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            filtered_events=set(),
+            filtered_appearance_freqs={},
+            node_sizes={},
+            node_stats_map={},
+        )
+        graph = self.bpmn_converter.build_inductive_graph(empty_data)
+
+        self.assertIn("Start", graph.graph.source)
+        self.assertIn("End", graph.graph.source)
+        self.assertIn(("Start -> End"), graph.graph.source)
```
Comment (fabianf00) [inline tests/converters/bpmn_converter_test.py]: Use `graph.contains_node` and `contains_edge` instead of searching for graphviz strings

**Review by fabianf00 — CHANGES_REQUESTED** [review]
Please update the assertion functions and add a `__init__.py` file inside `tests/converters`. Without it the tests are never found/executed

**Review by fabianf00 — APPROVED** [review]
